README.md

```diff
@@ -1,2 +1,48 @@
 # megafuel-go-sdk
-Golang SDK for the eoa-paymaster clients
+
+This is Golang SDK for the MegaFuel clients, it provides a simple way to interact with the MegaFuel.
+
+For more information, please refer to the [API documentation](https://docs.nodereal.io/docs/megafuel-api).
+
+## Quick Start
+1. Install dependency
+ ```shell
+ $ go get -u github.com/nodereal/megafuel-go-sdk
+ ```
+   
+2. Example
+```go
+package main
+
+import (
+	"context"
+	
+	"github.com/node-real/megafuel-go-sdk/pkg/paymasterclient"
+)
+
+const (
+    SPONSOR_URL = "https://sponsor-api.nodereal.io"
```

Comment:
wrong url.

Reply:
By the way, because we provide fixed endpoint for testnet and mainnet, can we define some const so that user dont have to input the URL by themselves?
Of cource, we can still allow develop to custom the endpoint

Reply:
as js-sdk do

Reply:
please fix

Reply:
fixed